### PR TITLE
zh-trans: /docs/reference/setup-tools/kubefed/kubefed-unjoin.md

### PR DESCRIPTION
**Pull request by AdamDang**
zh-trans: /docs/reference/setup-tools/kubefed/kubefed-unjoin.md

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+notitle: true
+weight: 50
+---
+## kubefed unjoin
+
+<!--
+Unjoin a cluster from a federation
+-->
+
+从联邦中取消集群连接
```
Comment (SataQiu) [inline content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md]:
```suggestion
从联邦中移除一个集群
```

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+<!--
+### Synopsis
+-->
+### 概要
+
+<!--
+Unjoin a cluster from a federation.
+-->
+
+从联邦中取消集群连接
```
Comment (SataQiu) [inline content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md]:
```suggestion
从联邦中移除一个集群
```

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+  # Unjoin the specified cluster from a federation.
+  # Federation control plane's host cluster context name
+  # must be specified via the --host-cluster-context flag
+  # to properly cleanup the credentials.
+  kubectl unjoin foo --host-cluster-context=bar --cluster-context=baz
+```
+-->
+
+```
+  # 从联邦中取消指定群集的连接。必须通过 --host-cluster-context 参数指定联邦控制平面的主机群集上下文名称，才能正确清理凭证。
```
Comment (SataQiu) [inline content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md]:
```suggestion
   # 从联邦中移除指定集群。必须通过 --host-cluster-context 参数指定联邦控制平面所在群集的上下文名称，以便正确清理凭证。
```

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+  -h, --help                                 help for unjoin
+      --host-cluster-context string          Host cluster context
+```
+-->
+
+```
+      --cluster-context string               本地 kubeconfig 中的集群上下文名称。 如果未指定，默认为集群名称。
+      --federation-system-namespace string   主机集群上用来安装联邦系统组件的命名空间（默认 "federation-system"）
+  -h, --help                                 unjoin 的帮助信息
+      --host-cluster-context string          主机集群的上下文
```
Comment (SataQiu) [inline content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md]:
```suggestion
      --host-cluster-context string          宿主集群的上下文
```

**Comment by SataQiu** [comment]
@AdamDang 一点建议，请参考!

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md (hunk 1 above)
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md]: Done！

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md (hunk 2 above)
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md]: Done！

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md (hunk 3 above)
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md]: Done！

File: content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md (hunk 4 above)
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubefed/kubefed-unjoin.md]: Done！

**Comment by chenrui333** [comment]
我看master branch 好像没有 `kubefed-unjoin.md` 了，其实可以考虑把这个 release-1.12 的页面也删掉？
screen shot 2019-01-06 at 1 30 33 am

**Comment by chenrui333** [comment]
Putting this PR on hold in favor of the newer one.
/hold

**Comment by chenrui333** [comment]
@AdamDang Let's merge these discussions into another similar PR work. 😄

/close

**Pull request closed without merging**